File: AlphaParking/AlphaParking.BLL/EventBus/IntegrationEvents/UserCreatedIntegrationEvent.cs
using AlphaParking.BLL.EventBus.Abstractions;
using AlphaParking.Models;

namespace AlphaParking.BLL.EventBus.IntegrationEvents
{
    public class UserCreatedIntegrationEvent: IntegrationEvent
    {
        public User User { get; set; }

        public UserCreatedIntegrationEvent(User user)
            => User = user;
    }
}

File: AlphaParking.Java.AuthService/javaauth/src/main/java/com/auth/event_bus_utils/integration_events/UserCreatedIntegrationEvent.java
package com.auth.event_bus_utils.integration_events;

import com.auth.models.User;

public class UserCreatedIntegrationEvent extends IntegrationEvent{
    private User user;

    public UserCreatedIntegrationEvent(User user) {
        this.setUser(user);
    }

    /**
     * @return the user
     */
    public User getUser() {
        return user;
    }

    /**
     * @param user the user to set
     */
    public void setUser(User user) {
        this.user = user;
    }

}